pkg/provider/provider.go

```diff
@@ -289,6 +289,11 @@ func isSkipHelmChart(helmName string, skipHelmChartList []configuration.SkipHelm
 	return false
 }
 
+func ReloadTestEnvironment() TestEnvironment {
```

Comment:
I'm not sure whether this function is actually needed as we already have the `env.SetNeedsRefresh()`. We already use it in some of the lifecycle TS's test cases.

Reply:
Oh nice, I'll remove my func then and use the existing.